package.json

```diff
@@ -54,9 +61,12 @@
     "dependencies": {
         "howler": "^2.2.3"
     },
+    "resolutions": {
+        "typescript": "5.0.4"
```

Comment:
this ensures the right typescript version is used for scripts (tsdx comes with its own ts version which is old)